src/main/java/seedu/address/model/room/RoomNumberOrTagsContainsKeywordsPredicate.java

```diff
@@ -0,0 +1,29 @@
+package seedu.address.model.room;
+
+import java.util.List;
+import java.util.function.Predicate;
+/**
```

Comment:
Should there be a space between the bottom-most `import` and the start of the class header?

Reply:
Done